internal/controller/cluster_monitoring_controller.go

```diff
@@ -88,23 +91,27 @@ func (r *ClusterMonitoringReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	ctx = log.IntoContext(ctx, logger)
 
 	if !r.MonitoringConfig.Enabled {
```

Comment:
Do we still need this condition since you add `if !r.MonitoringConfig.IsMonitored(cluster)` ?

Reply:
Yes because we need to create/delete things if the installation is monitored (mimir secrets for instance) and some we need to do for all clusters

Reply:
This condition is covered by this one https://github.com/giantswarm/observability-operator/pull/48/files#diff-1a9ac66aa1d96d275ecfebdf5b5b47a1659ffe91674fe1fea9892587c8801f13R97
No ?

Reply:
Yes but this is an info log only. The issue is that thé mc cluster Can have thé giantswarm.io/monitoring: false but not the installation

Reply:
I'd have preferred adding the log in the `IsMonitored` function and keep only one condition in the controller.

Reply:
But I don't want the code to log everytime I use this condition :D

Reply:
And as mentionned, we cannot keep only 1 condition because we create MC level resources (like the mimir ingress) and resources for each WCs and they do not depend on the same value